packages/as-types/assembly/address.ts

```diff
@@ -0,0 +1,50 @@
+import { addressToBytes, bytesToAddress } from './serialization';
+
+/**
+ * A Massa's blockchain address.
+ *
+ */
+export class Address {
```

Comment:
Is it a good idea to bring Address into as-types? as-types is intended to be used in another context than massa sc dev.

Reply:
Address serialization will be done later